src/app/course/services/CoursesService.ts

```diff
@@ -0,0 +1,39 @@
+// @ts-ignore
```

Comment:
check file naming convention, please. You will get a mess in names if you won't use it.

Reply:
fix tsconfig.json